### PR TITLE
Khai resubmit

### PR DESCRIPTION
**Pull request by KhaiTPhan**
Updating search.js and api-routes.js with axios code.

### CODE REVIEW

File: public/js/search.js [hunk 1]
```diff
@@ -70,34 +74,49 @@ function displayMovieInfo() {
...
-          if (response.length = 0)
-          fetch('/api/movies', {
-            method: 'POST',
-            headers: {
-              'Content-Type': 'application/json',
-            },
-            body: JSON.stringify(review),
-          })
         })
+        // .then((response) => {
```
Comment (malakhim) [inline public/js/search.js]: I suggest you just delete this, as it doesn't serve any purpose any more

File: public/js/search.js [hunk 2]
```diff
@@ -70,34 +74,49 @@ function displayMovieInfo() {
...
+    displayMovieInfo(movie)
 
 })
 
 //On click event listener for movie buttons
 $(document).on("click", ".movie-btn", function () {
-    movie = $(this).attr("data-Title");
-    displayMovieInfo()
+    const movie = $(this).attr("data-Title");
+    displayMovieInfo() // should navigate to the reviews page with the selected movie
```
Comment (malakhim) [inline public/js/search.js]:
```suggestion
    displayMovieInfo() // TODO: should navigate to the reviews page with the selected movie
```

File: server.js [hunk 3]
```diff
@@ -22,6 +22,37 @@ require('./routes/html-routes.js')(app);
...
 
+
+
 db.sequelize.sync({ force: false }).then(() => {
   app.listen(PORT, () => console.log(`Server listening on: http://localhost:${PORT}`));
-});
\ No newline at end of file
+});
+
+// db.sequelize.sync().then(
```
Comment (malakhim) [inline server.js]: Might want to get rid of this too, as it's just dead code

**Review by norbu-s — APPROVED** [review]

File: public/js/search.js [hunk 4]
```diff
@@ -70,34 +74,22 @@ function displayMovieInfo() {
...
+    displayMovieInfo(movie)
 
 })
 
 //On click event listener for movie buttons
 $(document).on("click", ".movie-btn", function () {
-    movie = $(this).attr("data-Title");
-    displayMovieInfo()
+    const movie = $(this).attr("data-Title");
+    displayMovieInfo() // TODO: should navigate to the reviews page with the selected movie
```
Comment (malakhim) [inline public/js/search.js:92]: This needs a parameter

Comment (malakhim) [reply public/js/search.js:92]: `$(document).on("click", ".movie-btn", function myFunction() {
    location.replace("/addreview/" + movie.title);
});`

Comment (malakhim) [reply public/js/search.js:92]: Confirm that `movie.title` is the correct property

File: routes/api-routes.js [hunk 5]
```diff
@@ -1,9 +1,37 @@
...
+  
+  // GET route for getting movie titles
+  app.get('/api/search/:title', (req, res) => {
+    db.Movies.findAll({}).then((movies) => {
+      if (movies.length === 0){
+       
+       axios.get("https://www.omdbapi.com/?t=" + req.params.title + "&apikey=trilogy").then(omdbdata => {
+      // TODO: insert into movies database using sequelize here
+      db.Movies.create({
+        title: movie.title,
```
Comment (malakhim) [inline routes/api-routes.js:17]: movie is not defined here

Comment (malakhim) [reply routes/api-routes.js:17]: Suggest this line above db.Movies.create: `const movie = omdbdata[0]`;

**Pull request merged**